api/python/indigo/ml/clustering.py

```diff
@@ -0,0 +1,19 @@
+from sklearn.cluster import SpectralClustering
+
+
+def clustering(
+    assay_values: list[list], method=SpectralClustering, **kwargs
+) -> list:
```

Comment:
Please specify output list type, I guess it should be `list[int]` in this case.

Reply:
Done